src/main/resources/application.conf

```diff
@@ -57,6 +57,11 @@ etc-client {
   }
 
   blockchain {
+    frontier-block-number = "0"
+    homestead-block-number = "1150000"
+    eip150-block-number = "2500000"
+    eip160-block-number = "3000000"
```

Comment:
How would we configure our private net to start with the latest version? Would all the numbers, except for `eip160-block-number`, have to be negative?